fastlane/spec/cli_tools_distributor_spec.rb

```diff
@@ -43,6 +43,8 @@
     end
 
     it "checks for updates even if the lane has an error" do
+      allow(FastlaneCore::Env).to receive(:truthy?).and_return(:default)
+      allow(FastlaneCore::Env).to receive(:truthy?).with('FASTLANE_SKIP_DOCS').and_return(true)
```

Comment:
There is 3 duplication codes, so it is better to define helper in `fastlane_core/spec/spec_helper.rb` or `fastlane/spec/spec_helper.rb`?